source/docs/use/troubleshooting/diagnostics.md

```diff
@@ -1,7 +1,7 @@
 ---
-id: diagnostics
 title: Diagnostics
 description: 'How to get HACS diagnostics'
+icon: material/hammer-wrench

```

Comment:
These 2 should not have the same icon. If we can not find differentiating icons here, I would rather we did not have it in this section at all

Reply:
the idea was to have the same icon for all troubleshooting topics